plat/common/x86/cpu_native.c

```diff
@@ -46,7 +46,18 @@ unsigned long read_cr2(void)
 	return cr2;
 }
 
-void system_off(void)
+#ifdef CONFIG_KVM_VMM_QEMU
+/**
+ * Trigger an exit() in QEMU with the code `value << 1 | 1`.
+ * @param value the value used in the calculation of the exit code
+ */
+static void qemu_debug_exit(int value)
+{
+	outw(0x501, value);
```

Comment:
Would it make sense to have a macro for `0x501` - this doesn't tell me anything?